strapi/strapi
# ML: Use formatjs inline plural API, display folder count

### What does it do?

Updates the ML subtitle to use the formatjs inline plural API. It also starts to add a static folders count.

<img width="316" alt="Screenshot 2022-03-21 at 14 07 41" src="https://user-images.githubusercontent.com/2244375/159267219-d5fd7a83-e58d-4234-81cb-f8894416eeda.png">

### Why is it needed?

Less translation strings, more context.

### How to test it?

1. Go to the media library
2. See the subtitle
